spec/controllers/linked_data_terms_controller_spec.rb

```diff
@@ -390,48 +390,48 @@
     context 'producing internal server error' do
       context 'when server returns 500' do
         before do
-          stub_request(:get, 'http://localhost/test_default/term?uri=http://test.org/530369').to_return(status: 500)
+          stub_request(:get, 'http://localhost/test_default/term?uri=http://id.worldcat.org/fast/530369').to_return(status: 500)
```

Comment:
The worldcat uri is what is really in the data.  The test.org uri doesn't exist in the graph.  When working with ldpath, the data needs to be in the graph.